src/main/java/frc/robot/Constants.java

```diff
@@ -31,7 +31,7 @@
     public static final int flywheelLeftPort = 5;
     public static final int flywheelRightPort = 6;

```

Comment:
Can we update the comments or change `flyWheelRightPort` so we don't accidently run the shooter or smth?